data_hub_api/docmaps/v2/codecs/elife_manuscript.py

```diff
@@ -128,8 +129,18 @@ def get_elife_manuscript_part_of_section_complement(
                 ),
                 'thumbnail': related_content['collection_thumbnail_url']
             }
+        if related_content['podcast_id']:
+            podcast_dict = {
+                'type': 'Podcast',
+                'url': (
+                    'https://elifesciences.org/podcast/episode'
+                    + str(related_content['podcast_id'])
+                ),
+                'title': related_content['podcast_title'],
+                'description': related_content['podcast_desc']
+            }
         return remove_key_with_none_value_only(
-            [related_article_dict, collection_dict]
+            [related_article_dict, collection_dict, podcast_dict]
```

Comment:
I can see this restricts it to one of each type. I thought at least for related article we may have multiple articles?

Reply:
There is no example yet and I thought we may have not just multiple articles, multiple podcasts or multiple collections as well. It is a different task. Not added in this PR as it cover all others too.